.env

```
@@ -0,0 +1,3 @@
+MYSQL_ROOT_PASSWORD=wptmdgus03
+MYSQL_USER=jsh03
+MYSQL_PASSWORD=wptmdgus03
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)